sdk/storage/azure-storage-blob/swagger/README.md

```diff
@@ -16,7 +16,7 @@ autorest --java --use:@autorest/java@4.0.x
 
 ### Code generation settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/seanmcc-msft/azure-rest-api-specs/95d8b1a4c972eafa176dce585c45e7e0611fb04e/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-04-10/blob.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-04-10/blob.json
```

Comment:
Please check with @seanmcc-msft @rickle-msft @ibrahimrabab.

Reply:
I blindly changed this to main as Autorest wouldn't run as previous Swagger didn't contain some features, this was just to get this working

Reply:
We've been using `main` branch for most autorest input-file locations but other languages use a specific commit id to ensure regeneration doesn't automatically pull in new changes in the main branch. We should also start using commit ids here.

Reply:
I'll use what .NET: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e515b6251fdc21015282d2e84b85beec7c091763/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-04-10/blob.json